packages/xo-server/src/xapi-object-to-xo.js

```diff
@@ -586,9 +586,8 @@ const TRANSFORMS = {
       other_config: {
         ...obj.other_config,
         automatic:
-          obj.other_config?.automatic !== undefined
-            ? obj.other_config.automatic === 'true'
-            : null,
+          obj.other_config?.automatic !== undefined &&
+          obj.other_config.automatic === 'true',
```

Comment:
Once more :wink:

Reply:
I don't have the automatisms with the question marks yet. thanks.

Reply:
```
(A => B) => ( A∧B ⇔ A ) 
```